packages/node_modules/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -129,6 +130,28 @@ const Credentials = WebexPlugin.extend({
     /* eslint-enable camelcase */
   },
 
+  /**
+   * Extract the OrgId [realm] from a provided JWT.
+   *
+   * @param {string} token - The JWT to extract the OrgId from.
+   * @returns {string} - The OrgId.
+   */
+  extractOrgIdFromJWT(token = '') {
+    // Split the provided JWT into subsections.
+    const fields = token.split('.');
+
+    // Validate that the provided token is a JWT.
+    if (fields.length !== 3) {
```

Comment:
Is there another way to verify that something is a JWT? This check feels a little brittle (from my limited JWT knowledge)

Reply:
In this case, the above works well due to how `node-jose` works. If the provided token has three `.` characters, `node-jose` will interpret it. If the interpreted object doesn't have a `realm` key:value`, it returns undefined [as intended].

Reply:
In this case, the above works well due to how `node-jose` works. If the provided token has three `.` characters, `node-jose` will interpret it. If the interpreted object doesn't have a `realm` key:value`, it returns undefined [as intended].

Reply:
Since we are using the `jsonwebtoken` package elsewhere in the SDK, let's use it's [decode](https://www.npmjs.com/package/jsonwebtoken#jwtdecodetoken--options) method.

Reply:
addressed this.